.gitignore

```diff
@@ -63,4 +63,7 @@ typings/
 .yarn-integrity
 
 # next.js build output
-.next
\ No newline at end of file
+.next
+
+# commit message template
+.msg
```

Comment:
Should not be a part of this PR

Reply:
it is for entering detailed commit message